src/integrations/third-party/the-events-calendar/EventsCalendarSchema.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party\The_Events_Calendar
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
```

Comment:
Given this is a schema piece it should be located along the other schema pieces.

I would suggest `src/generators/schema/third-party`.
```suggestion
namespace Yoast\WP\SEO\Generators\Schema\Third_Party;
```